components/Lead.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div>
+    <div class="lead">
+      <h2 class="lead__title">{{ content.title }}</h2>
+      <p class="lead__dates">{{ content.dates }}</p>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    content: {
+      type: Object,
+      required: true,
+    },
+  },
+}
+</script>
+
+<style lang="scss" scoped>
+.lead {
+  margin-top: 80px;
+  @media screen and (min-width: $screen-ipad) {
+    margin-top: 160px;
+  }
+  &__title {
+    @include mobileToDesktopFontSize(var(--fs-large), var(--fs-giant));
+    @include gradientTitle(var(--bg-gradient-start), var(--bg-gradient-end));
+
+    max-width: 1024px;
+    margin-bottom: 0;
+    color: var(--fc-default);
+    font-weight: var(--fw-bold);
+    line-height: 1;
```

Comment:
I'm a css noob but isn't it better to use explicit units except for `0`? I'm not sure so it is a good opportunity to learn from you 😄

Reply:
On the specific case of `line-height` the value it's unitless since acts as a multiplier for its own `font-size` so it's preferred to set it without units to avoid unexpected computed results.

Reply:
Oh, damn I don't know how to read a comment, you're completely right @beagleknight the `margin` property should be `0px` sorry for the misunderstanding!